modules/ROOT/pages/create-profiles.adoc

```diff
@@ -12,38 +12,67 @@ The API Governance console UI guides you through creating profiles.
 
 . In *API Governance*, click *New Profile*.
 . Follow the UI prompts to enter the following:
-+
-* *General Information:* Enter a profile name and description that helps others understand what kinds of governance rulesets are included and the kinds of APIs that are validated in this profile.
-* *Rulesets*: Select one or more rulesets you want to use to govern your APIs.
-+
-** Use the search box and filters to find the rulesets you want to use. You can filter rulesets by the following:
-+
-*** Organization
-*** Tag
-*** Catalog
-+
-** You can select the version of each ruleset you want to use or select the *Latest* option. The default is *Latest*. 
-+
-*** If you select *Latest*, the profile automatically uses the latest version of that ruleset if any versions are published after the profile is saved. This ensures that the APIs selected in this profile are always validated against the latest version of rulesets without the need to manually update the profile. 
-*** If you select a specific version, the profile continues to use that version even if new versions of the rulesets are published after the profile is saved. This ensures that APIs selected in this profile that are conformant to a specific ruleset version are not inadvertently made nonconformant by a newly published version of the ruleset. You can update the profile and select a new version when you are ready to use the new ruleset version.
-
-* *Filter Criteria*: To apply selected rulesets to specific APIs, select filter criteria. The filter criteria applies to your organization's APIs that are in Exchange and to new APIs as you add them to Exchange. You can filter by the following:
-+
- ** API type
- ** Tags
- ** Categories
- ** Lifecycle State
-+
-Selections appear in the *Tag* and *Category* fields only if your APIs in Exchange have tags and categories. If you filter by multiple tags and categories, your APIs must have all of the selected tags and categories to match the criteria. See xref:add-tags.adoc[Identifying APIs to Govern].
-+
-Based on your filter criteria, the latest version of the APIs are automatically selected and show in the page. For example, if you select lifecycle state *Development*, the latest versions of the APIs in *Development* are selected rather than the latest stable versions. 
-+ 
-If you filter by categories, you can see the number of categories for each API, which is denoted by *Categories* followed by a number in parenthesis. Hover over *Categories* to see more details. 
-+
+
+[cols="1a,9a"]
+|===
+
+General Information 
+|Enter a profile name and description that helps others understand what kinds of governance rulesets are included and the kinds of APIs that are validated in this profile.
+| 
+Rulesets
+|
+Select one or more rulesets you want to use to govern your APIs.
+
+Use the search box and filters to find the rulesets you want to use. You can filter rulesets by the following:
+
+* Organization
+* Tag
+* Catalog
```

Comment:
*Categories

Reply:
Actually, @sandrau, in QAX I only see organization and tag. Did they decide not to filter by category?

Reply:
Oh, nvm, I see that the filters appear only if you have a ruleset that fits that filter. I'll add that to the doc.